discord/opus.py

```diff
@@ -57,6 +64,15 @@ class SignalCtl(TypedDict):
     music: int
 
 
+class EncoderKwargs(TypedDict, total=False):
+    application: APPLICATION_CTL
+    bitrate: int
+    fec: bool
+    packet_loss_pct: float
```

Comment:
```suggestion
    expected_packet_loss: float
```

Shortcuts in typing are usually not done unless necessary and this one's better off longer.